data_management/autopreprocess_testing/preprocess_directory.py

```diff
@@ -1,34 +1,46 @@
 """
 Base code for batch pre-processing
 1. Gets file from directory
-2. Does some pre-processing on it
+2. Does pre-processing on the file
 3. Puts processed file in new directory
 """
-
+import sys
+import queue
 import cv2
 import os
 import glob
 from PIL import Image
-from processing_pipeline import processing_pipeline
-
-# Change to current directory
-os.chdir(os.path.dirname(os.path.abspath(__file__)))
-# Test image directory
-source_dir = os.getcwd() + '/test_images'
-# Processed image directory
-target_dir = os.getcwd() + '/processed_test_images/'
+from processing_pipeline import processing_pipeline 
 
 # Get list of all image paths
-supported_file_extensions = ["jpg", "jpeg", "png", "tif", "tiff"]
-img_files = []
-for ext in supported_file_extensions:
-    img_files += glob.glob(source_dir + f"/**/*.{ext}", recursive=True)
+def get_all_images(source_dir):
+
+    supported_file_extensions = ["jpg", "jpeg", "png", "tif", "tiff"]
+    img_files = []
+    for ext in supported_file_extensions:
+        img_files += glob.glob(source_dir + f"/**/*.{ext}", recursive=True)
+    return img_files
+
+# Test image directory
+# Change to current directory
+default_directory = os.path.join(os.path.dirname(os.path.abspath(__file__)),'test_images22622')
+if len(sys.argv) > 1:
+    default_directory = sys.argv[1]
+source_dir = default_directory
+# Processed image directory, I also changed the file folder, user needs to change the folder to the one they wish to train images on.
+target_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)),'processed_test_images')
+# cwd = os.getcwd()
```

Comment:
delete line 32 to 34 of dead code

Reply:
The dead code in lines 32 to 34 have been deleted.